app/components/ui/atoms/StickyFooter.md

```diff
@@ -3,18 +3,24 @@ Footer which is stuck to the bottom of the viewport.
 ...look to the bottom of the viewport to see this component :)
 
 ```js
+initialState = { show: false }
 const Button = require('@pubsweet/ui').Button
 const { Flex, Box } = require('grid-styled')
-;<StickyFooter>
-  <Flex>
-    <Box mr={3}>
-      <Button onClick={() => console.log('cancelled')}>Never mind</Button>
-    </Box>
-    <Box>
-      <Button primary onClick={() => console.log('accepted')}>
-        Click on me!
-      </Button>
-    </Box>
-  </Flex>
-</StickyFooter>
+;<div>
+  <button onClick={() => setState({ show: !state.show })}>Toggle</button>
```

Comment:
Might be worth changing the text that describes this component. Right now it says '....look to the bottom of the viewport to see this component', which is no longer the case.

Could also be good to make it really clear (without needing to first look at the code) that this button has nothing to do with the `StickyFooter` itself.

Reply:
@jsms90 .. it works for me. It toggles the `StickyFooter` at the bottom of the viewport :thinking: